sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallAutomationLiveTestBase.java

```diff
@@ -69,7 +69,7 @@ public class CallAutomationLiveTestBase extends TestProxyTestBase {
     protected static final String PHONE_USER_1
         = Configuration.getGlobalConfiguration().get("TARGET_PHONE_NUMBER", "+16471234567");
     protected static final String MEDIA_SOURCE
-        = Configuration.getGlobalConfiguration().get("ACS_MEDIA_SOURCE", "https://contoso.com/music.wav");
+        = Configuration.getGlobalConfiguration().get("FILE_SOURCE_URL", "https://contoso.com/music.wav");
```

Comment:
Why do we need to change this?

Reply:
This matches with the same format as other 3 SDKs. All 3 SDK uses "FILE_SOURCE_URL" as file source, but Java for some whatever reason uses this different name. This is to uniform as much as possible between SDK live test to reduce pain to have different treatment of each SDK live tests